add CAL_STATIC_CAPABILITY_NO_SEND_IMIP calendar capability

From: Ondrej Jirman <ondrej.jirman@zonio.net>

calendar/ChangeLog:

2008-04-06  Ondrej Jirman  <megous@megous.com>

	** Fix for bug #not yet submitted

	* libecal/e-cal-util.h:
	Add new capability CAL_STATIC_CAPABILITY_NO_SEND_IMIP to disable sending
	of iMIPs and conversion of attachment URLs to 'CID:...' by evolution.

	This is used by calendar backends that want to send iTIPs and upload
	attachments to a calendar server.
---

 calendar/libecal/e-cal-util.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)


diff --git a/calendar/libecal/e-cal-util.h b/calendar/libecal/e-cal-util.h
index 0be3bd4..8cf10f6 100644
--- a/calendar/libecal/e-cal-util.h
+++ b/calendar/libecal/e-cal-util.h
@@ -126,6 +126,7 @@ gboolean e_cal_util_event_dates_match (icalcomponent *icalcomp1, icalcomponent *
 #define CAL_STATIC_CAPABILITY_NO_ORGANIZER		 "no-organizer"
 #define CAL_STATIC_CAPABILITY_DELEGATE_TO_MANY		 "delegate-to-many"
 #define CAL_STATIC_CAPABILITY_HAS_UNACCEPTED_MEETING     "has-unaccepted-meeting"
+#define CAL_STATIC_CAPABILITY_NO_SEND_IMIP		 "no-send-imip"
 
 /* Recurrent events. Management for instances */
 icalcomponent *e_cal_util_construct_instance (icalcomponent *icalcomp,
